src/_ert_forward_model_runner/reporting/message.py

```diff
@@ -119,3 +119,10 @@ class Exited(Message):
     def __init__(self, job, exit_code):
         super().__init__(job)
         self.exit_code = exit_code
+
+
+class Manifest(Message):
+    def __init__(self, data: Dict[str, Any], run_path: str):
```

Comment:
The word `data` says very little here. I would suggest creating a TypeDict for it in `src/_ert` and calling it `ManifestDict`:

```python

class ManifestDict(TypedDict):
    type: Literal["file"]
    path: str

class Manifest(Message):
    def __init__(self, manifest_dicts: Dict[str, ManifestDict], run_path: str):
        ...
```